### ISSUE TITLE
Unable to Install piper-tts on macOS Due to piper-phonemize 1.1.0 Dependency

### ISSUE THREAD

**Issue by gabrielgq** [rhasspy/piper#217]
#### Description

I am trying to install `piper-tts` on macOS. However, the installation process fails because of its dependency on `piper-phonemize` version 1.1.0, which appears to lack a macOS-compatible distribution.

#### Steps to Reproduce

1. On a macOS system, execute: `pip install piper-tts`
2. Installation fails due to the dependency on `piper-phonemize` 1.1.0.
3. Direct installation of `piper-phonemize` 1.1.0 also fails: `pip install piper-phonemize==1.1.0`

#### Expected Behavior

I anticipated that `piper-tts` would be installed successfully on my macOS system without issues related to its dependencies.

#### Actual Behavior

The installation process is halted because there isn't a macOS-compatible wheel or source distribution available for `piper-phonemize` version 1.1.0.

#### Additional Information

When inspecting the [PyPI page for `piper-phonemize` 1.1.0](https://pypi.org/project/piper-phonemize/1.1.0/#files), it's evident that wheels are available for Linux (`manylinux`) for Python versions 3.9, 3.10, and 3.11, but no distributions are present for macOS.

#### Questions

1. Is the lack of a macOS distribution for `piper-phonemize` 1.1.0 a known issue?
2. Are there plans to release a macOS-compatible wheel or source distribution for this version?
3. As I am keen to use `piper-tts` on macOS, is there a recommended workaround or alternative version that I can utilize in the interim?

Thank you for your prompt attention to this matter!

**Comment by wdebusschere** [comment]
@gabrielgq I have exact the same problem.

**Comment by Salvakiya** [comment]
same problem with python 3.11.4 on windows 11 x64

**Comment by nafets33** [comment]
same here, I attempted docker image but also ran into issues : (

**Comment by tuxpoldo** [comment]
Maybe related to rhasspy/piper-phonemize#11 ? When merged, check if this fixes the issue.

**Comment by eshack94** [comment]
Seeing the same issue on MacOS.

**Comment by Sendery** [comment]
Seeing the same issue on MacOS when installing with pip.. I'm going to try it compiling from source

**Comment by williamcorney** [comment]
@Sendery   Any luck ?

Same problem.  I will post if I can figure out a solution

**Comment by rmcpantoja** [comment]
Piper phonemize is already for Windows and mac but we only need a Python wheel. I tried to build it but failed.

**Comment by Aptronymist** [comment]
Yeah, same on my windows. A shame, I really wanted to try this out!

**Comment by rmcpantoja** [comment]
Hi @Aptronymist, 
you can try without Python API, just download the last asset in the releases page.

**Comment by Aptronymist** [comment]
> Hi @Aptronymist, 
> you can try without Python API, just download the last asset in the releases page.

Oh I got that , just wanted to be able to use it with python too.

**Comment by Sendery** [comment]
@williamcorney I did compile it, or I think... 
I did have to download the piper-phonemize into the lib folder. I download a few architectures (macos-aarch64 and Linux-arm64)
The compilation work and it generated a few executables in the build folder, it even past the single test during compilation.
The piper executable wasnt install in any "bin" or "script" folder of the system that i notice 
I did continue anyway and downloaded a few models and try the piper executable with the "echo .... | piper ... " command
When executing the piper executable in the build folder it reports:
```/usr/share/espeak-ng-data/phontab': No such file or directory ```
But that folder was used during compilation from:
```piper/install/espeak-ng-data/phondata ```
I did try to link, copy and move... to that path the folder, but I couldnt due to MacOS folder system (or thats what i thought, Im not very expert on MacOS)
And I dont know if I can set piper to use anohter path.

I install espeak-ng with ports but didnt do the trick either.

For the moment I'm stuck

**Comment by kerwinkfs** [comment]
Same problem with my man as well.

**Comment by nickolay** [comment]
I posted steps that worked for me to https://github.com/rhasspy/piper-phonemize/issues/14

**Comment by willwade** [comment]
if you need to work around this download the wheel: https://github.com/rhasspy/piper-phonemize/issues/14#issuecomment-2094395478 so 

``pip install piper_phonemize-1.1.0-cp311-cp311-macosx_11_0_arm64.whl
pip install piper-tts``

**Comment by jtoy** [comment]
I also can't get it working on mac:
The conflict is caused by:
    piper-tts 1.2.0 depends on piper-phonemize~=1.1.0
    piper-tts 1.1.0 depends on piper-phonemize~=1.0.0

**Comment by jdeltoft** [comment]
You might try using pipx. I followed this approach after seeing it mentioned in the error the Mac reports. I did this and then had no issues using piper in python as listed in piper's README 

I also read up a bit here: https://mac.install.guide/python/pipx

<img width="605" alt="image" src="https://github.com/user-attachments/assets/781495cf-f436-431f-9cb5-8680268193b0">

**Comment by ufodriverr** [comment]
Is this project like dropped?
Cos i have same issue:
```
Collecting piper-tts
  Downloading piper_tts-1.2.0-py3-none-any.whl.metadata (776 bytes)
INFO: pip is looking at multiple versions of piper-tts to determine which version is compatible with other requirements. This could take a while.
  Downloading piper_tts-1.1.0-py3-none-any.whl.metadata (776 bytes)
ERROR: Cannot install piper-tts==1.1.0 and piper-tts==1.2.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    piper-tts 1.2.0 depends on piper-phonemize~=1.1.0
    piper-tts 1.1.0 depends on piper-phonemize~=1.0.0
```

And when i try to install piper-phonemize i get:
```
ERROR: Could not find a version that satisfies the requirement piper-phonemize (from versions: none)
ERROR: No matching distribution found for piper-phonemize
```

**Comment by ehsanul** [comment]
This worked for me:

```sh
pip install piper-tts --no-deps piper-phonemize-cross onnxruntime numpy
```

Source: https://github.com/rhasspy/piper/issues/509#issuecomment-2575654706
